src/ui_widgets/ContextPopup.gd

```diff
@@ -217,11 +220,30 @@ min_width := -1.0, separator_indices := PackedInt32Array()) -> void:
 			main_container.add_child(_setup_button(buttons[idx], align_left))
 		if min_width > 0:
 			custom_minimum_size.x = min_width
+		if max_height > 0 and max_height < get_minimum_size().y:
+			custom_minimum_size.y = max_height
+			main_container.get_parent().vertical_scroll_mode = ScrollContainer.SCROLL_MODE_AUTO
+
 
 # Helper.
 func _common_initial_setup() -> VBoxContainer:
+	# Create a ScrollContainer to allow scrolling.
+	var scroll_container := ScrollContainer.new()
+	scroll_container.horizontal_scroll_mode = ScrollContainer.SCROLL_MODE_DISABLED
+	scroll_container.vertical_scroll_mode = ScrollContainer.SCROLL_MODE_DISABLED
+	
+	# Increase the scrollbar area on Android.
+	if OS.get_name() == "Android":
+		var scrollbar := scroll_container.get_v_scroll_bar()
+		var stylebox := scrollbar.get_theme_stylebox("scroll").duplicate()
+		stylebox.content_margin_left = 10
+		stylebox.content_margin_right = 10
+		scrollbar.add_theme_stylebox_override("scroll", stylebox)
```

Comment:
I think it would make more sense to add this as a theme variation inside ThemeUtils, or even have a condition for the default scroll styleboxes to make the content margin bigger on Android.

Reply:
The scrollbar is only needed for these popup panels, as they contain buttons, making touch scrolling support tricky. Everywhere else, touch scrolling works perfectly fine. That's why I made a direct modification here.

Reply:
Hmm... okay, we can do that if the pattern appears again.